mmv1/third_party/terraform/services/bigquery/resource_bigquery_table_test.go.tmpl

```diff
@@ -45,6 +45,43 @@ func TestAccBigQueryTable_Basic(t *testing.T) {
 	})
 }
 
+func TestAccBigQueryTable_DataPolicies(t *testing.T) {
+	t.Parallel()
+
+	datasetID := fmt.Sprintf("tf_test_%s", acctest.RandString(t, 10))
+	tableID := fmt.Sprintf("tf_test_%s", acctest.RandString(t, 10))
+	dataPolicyID := fmt.Sprintf("tf_test_data_policy_%s", acctest.RandString(t, 10))
+	dataCatTaxonomy := fmt.Sprintf("tf_test_taxonomy_%s", acctest.RandString(t, 10))
+	projectID := envvar.GetTestProjectFromEnv()
+	dataPolicyName := fmt.Sprintf("projects/%s/locations/us-central1/dataPolicies/%s", projectID, dataPolicyID)
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckBigQueryTableDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccBigQueryTableDataPolicies(datasetID, tableID, dataPolicyID, dataCatTaxonomy, dataPolicyName),
+			},
+			{
+				ResourceName:            "google_bigquery_table.test",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"deletion_protection", "ignore_schema_changes"},
+			},
+			{
+				Config: testAccBigQueryTableUpdated(datasetID, tableID),
```

Comment:
The only differences between this config and the previous config should be in `dataPolicies`.

Then we can check that no diffs in terraform plan with `ExpectNonEmptyPlan: false`.
Example: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/artifactregistry/resource_artifact_registry_repository_test.go.tmpl#L105-L108